scripts/linting/mdn-sitemap-compare.js

```diff
@@ -0,0 +1,130 @@
+/*
+A linter that can check that all of our URLs match URLs in MDN's sitemaps.
+
+Usage:
+
+npm run mdn-sitemap-compare [locale [locale2]]
+
+*/
+
+const url = require("url");
+const fetch = require("node-fetch");
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+
+const readFile = util.promisify(fs.readFile);
+
+const baseURL = "https://developer.mozilla.org";
+
+const ROOT = path.join(__dirname, "..", "..");
+
+async function fetchSitemap(url) {
+    if (url.startsWith("/")) url = `${baseURL}${url}`;
+    let r = await fetch(url);
+    let payload = await r.text();
```

Comment:
I don't know how much you care about nitpicky comments, or what our policy is on this for that matter, but this and the previous line could be `const`.